frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -0,0 +1,155 @@
+import { useNavigate } from 'react-router-dom';
+import { useLoad } from '@/hooks/api';
+import * as exchangesAPI from '@/api/exchanges';
+import { useContext, useEffect, useState } from 'react';
+import { ExchangeSelectionRadio } from './exchangeselectionradio';
+import { Button } from '@/components/forms/button';
+import { useTranslation } from 'react-i18next';
+import { getVersion } from '@/api/bitbox02';
+import style from '../exchange.module.css';
+import { AppContext } from '@/contexts/AppContext';
+import { TInfoContentProps } from './infocontent';
+
+type TProps = {
+  accountCode: string;
+  deviceIDs: string[];
+  selectedRegion: string;
+  onSelectExchange: (exchange: string) => void;
+  selectedExchange: string;
+  goToExchange: () => void;
+  showBackButton: boolean;
+  action: exchangesAPI.TExchangeAction
+  setInfo: (into: TInfoContentProps) => void;
+}
+
+export const BuySell = ({
+  accountCode,
+  deviceIDs,
+  selectedRegion,
+  onSelectExchange,
+  selectedExchange,
+  goToExchange,
+  showBackButton,
+  action,
+  setInfo,
+}: TProps) => {
+  const { t } = useTranslation();
+  const { setFirmwareUpdateDialogOpen } = useContext(AppContext);
+
+  const exchangeDeals = useLoad(() => exchangesAPI.getExchangeDeals(action, accountCode, selectedRegion), [action, selectedRegion]);
+  const [exchangeDealsError, setExchangeDealsError] = useState('');
+  const [updateFirmwareToSell, setUpdateFirmwareToSell] = useState(false);
+  // FIXME: we should make a BE `can-sell` endpoint
```

Comment:
don't let your dreams be dreams :stuck_out_tongue: 

Just add `supportsPocketPaymentRequest` or sth like that here: https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/39f90f84abb1326d208447a099a6bf27c607db2c/backend/devices/bitbox02/handlers/handlers.go#L304

Reply:
see my other comment below :)